ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -34,6 +34,9 @@ import SwiftUI
     /// The image used to fill the ring as a custom color.
     var imageBasedRingColor: UIImage? { get set }
 
+    /// An override for the template icon to use when there is no set image or name.
+    var customDefaultImage: UIImage? { get set }
```

Comment:
Does `Custom` really mean much here? Could it be simplified to just `defaultImage`? By setting it you are implying it is custom already?

Reply:
or at least for the modifier API, here it would still make sense to distinguish between variables.

Reply:
Yeah I went back and forth there. Due to the design of `MSFAvatar`, this name will have to match the public API for at least UIKit consumers.

I don't love having a variable named `defaultImage` that might be `nil` even when we clearly have a value, but it does match the rest of the API, so I guess we should be consistent.